Symmetry-Adapted-Perturbation-Theory/SAPT0_no_S2.py

```diff
@@ -0,0 +1,264 @@
+# A script to compute the SAPT0 interaction energy without the the Single-Exchange Approximation
+# 
+# Created by: Jonathan M. Waldrop
+# Date: 1/9/2018
+# License: GPL v3.0
+# 
+# Relevant equations can be found in the following papers:
+# Jeziorski, B.; Bulski, M.; Piela, L. Int. J. Quantum Chem. 1976, 10, 281-297.
+# Schaffer, R.; Jansen, G. Theor. Chem. Acc. 2012, 131, 1235-1244.
+# Schaffer, R.; Jansen, G. Mol. Phys. 2013, 111, 2570-2584.
+
+import psi4,time
```

Comment:
Can you run this whole thing through google's YAPF formatter? `YAPF` can be pip installed.